service/payment/api/api.go

```diff
@@ -21,7 +21,7 @@ import (
 	"os"
 	"strconv"
 
-	"github.com/labring/sealos/pkg/pay"
```

Comment:
services must not require other `services`/`controllers`

Reply:
Is there no solution to this problem?

Dockerfile doesn't seem to be able to introduce sealos/pkg